app/poros/daily.rb

```diff
@@ -0,0 +1,9 @@
+class Daily
```

Comment:
If not too much work, this could be renamed to something more clear like DailyForecast

Reply:
Sure thing

Reply:
Changing it now. I'll push in a bit

Reply:
Changed poro name from Daily to DairyForecast

Updated all dependencies and all tests are currently passing.